src/Faker/Provider/tr_TR/Person.php

```diff
@@ -39,37 +71,93 @@ class Person extends \Faker\Provider\Person
+        'Gül','Gülay','Gülce','Gülru','Gülsüm','Güntülü','Hacer','Hadra','Hafize',
+        'Halide','Halime','Halise','Hamdiye','Hamide','Hamra','Handan','Hande','Hanife',
+        'Hasibe','Hatice','Havva','Hayriye','Hazal','Hazel','Hicran','Hilal','Hira',
+        'Huban','Hulya','Hülya','Hümeyra','Idil','Ilay','Ilayda','Ilda','Ilgın','Ilgın','Ilsu','Ipek','Irem',
+        'Kadriye','Kamile','Kaniye','Kardelen','Karya','Kay','Kebire','Kevser',
+        'Kezban','Kübra','Lamia','Lara','Lena','Leya','Leyan','Leyla','Leyli',
+        'Marziye','Mavi','Maya','Mebruke','Mebrure','Mediha','Medine','Mehlika','Mehtap',
+        'Melda','Melek','Meliha','Melike','Melis','Melisa','Melissa',
+        'Meltem','Memduha','Mensure','Meray','Mercan','Merve','Meryem','Merzuka','Mesude','Meyra','Mihriban',
+        'Mina','Mine','Mira','Miran','Miya','Miyase','Muazzez','Munise','Müge',
+        'Naciye','Nafiye','Nagehan','Nahide','Naile','Naime','Narin','Naz','Nazan','Nazen',
+        'Nazenin','Nazife','Nazire','Nazmiye','Nebahat','Nebile','Necibe','Necla','Nehir',
+        'Nejla','Nergis','Neriman','Nermin','Nesibe','Nesli','Neslihan','Nesrin','Nesteren',
+        'Neval','Nevin','Nevriye','Neyra','Neyran','Nigar','Nihal',
+        'Nihan','Nil','Nila','Nilay','Nilsu','Niran','Nisa','Nisan','Nur','Nurbanu',
+        'Nuriye','Nursima','Nurten','Oya','Özlem','Pakize','Pamir','Pelin','Peri','Perihan',
+        'Perin','Perizat','Pervin','Peyker','Peyman','Piraye','Rabia','Rahil','Rahime','Raife',
+        'Raziye','Refia','Refika','Refiye','Remziye','Rengin','Reyhan','Reyya',
+        'Ruhsar','Rukiye','Ruziye','Rüveyda','Rüya','Sabriye','Sadberk','Sadiye','Safa',
+        'Safiye','Sahra','Sakine','Saliha','Sanem','Sar','Sara','Sare','Seba','Sebahat',
+        'Sebile','Seda','Sedef','Seden','Sefa','Seher','Selay','Selcan','Selda','Selen',
+        'Selime','Selin','Selma','Selmin','Selva','Selvet','Sema','Semanur','Semira','Semiye',
+        'Semra','Sena','Senem','Seniha','Sera','Serap','Seray','Seren','Serfiraz','Sermin',
+        'Serpil','Serra','Seval','Sevgi','Sevil','Sevilay','Sevim','Seyra','Sezin','Sibel',
+        'Sima','Simay','Simru','Sinem','Sinem','Songül','Suat','Sude',
+        'Sudiye','Sulbiye','Sultan','Sumru','Suna','Sunay','Suzan','Süeda','Sümeyra','Şah',
+        'Şahnur','Şerife','Şevval','Şeyda','Şeyma','Şule','Tanju','Tansel','Tansu',
+        'Tanzer','Tomris','Toprak','Tuba','Tuğçe','Tulca','Tulu',
+        'Tun','Tutku','Tuvana','Umut','Utku','Utkun','Ümran','Verda','Vildan','Vuslat','Yağmur',
+        'Yasemin','Yekta','Yelda','Yeliz','Yosma','Zade','Zahide','Zehra','Zeliha','Zerrin',
+        'Zeynep','Zuhal','Zülal','Züleyha','Zümra',
     );
 
     /**
      * @link http://tr.wikipedia.org/wiki/Kategori:T%C3%BCrk%C3%A7e_soyadlar%C4%B1
-     *
+     * @link http://www.nvi.gov.tr/Files/File/Istatistikler/Isimler/en%20%C3%A7ok%20kullan%C4%B1lan%20soyad.pdf
```

Comment:
Did you mix the previous names with this new source? I'd prefer that we keep only one.